ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -650,19 +659,28 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
-				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
mostly curious, why was this change needed?

Reply:
May be a relic of my insanity during testing